lib/os/zvfs/Kconfig

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2020 Tobias Svehagen
+# Copyright (c) 2023 Meta
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ZVFS
+	bool "Zephyr virtual filesystem (ZVFS) support [EXPERIMENTAL]"
+	select FDTABLE
```

Comment:
Probably need to `select EXPERIMENTAL` here

Reply:
Fixed!